src/js/renderers/dash.js

```diff
@@ -7,6 +7,9 @@ import {renderer} from '../core/renderer';
 import {createEvent} from '../utils/general';
 import {typeChecks} from '../utils/media';
 import {HAS_MSE} from '../utils/constants';
+import {loadScript} from '../utils/dom';
+
+let scriptPromise;
```

Comment:
By using the same scriptPromise we effectually use the same cache script loading